deploy/csi-operator.yaml

```diff
@@ -750,6 +750,10 @@ spec:
               # has been installed
             - name: OPENEBS_NAMESPACE
               value: openebs
+            - name: OPENEBS_IO_INSTALLER_TYPE
+              value: "cstor-operator"
+            - name: OPENEBS_IO_ENABLE_ANALYTICS
+              value: "true"
```

Comment:
let the default here.. for CI be set as false. When providing the YAML via fixed tags, this should be set to `true`. This will avoid CI/e2e images from being accounted.

Reply:
done